sycl/include/CL/sycl/detail/defines_elementary.hpp

```diff
@@ -0,0 +1,54 @@
+//==--- defines_elementary.hpp ---- Preprocessor directives (simplified) ---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#ifndef __SYCL_DISABLE_NAMESPACE_INLINE__
+#define __SYCL_INLINE_NAMESPACE(X) inline namespace X
+#else
+#define __SYCL_INLINE_NAMESPACE(X) namespace X
+#endif // __SYCL_DISABLE_NAMESPACE_INLINE__
+
+#ifndef __has_attribute
+#define __has_attribute(x) 0
+#endif
+
+#ifndef __has_builtin
+#define __has_builtin(x) 0
+#endif
+
+#if __has_attribute(always_inline)
+#define ALWAYS_INLINE __attribute__((always_inline))
+#else
+#define ALWAYS_INLINE
+#endif
+
+#ifndef __has_builtin
```

Comment:
__has_builtin is done 2x (see line 21), so this is redundant/meaningless.

Reply:
Sorry, I've missed that. Removed.